src/main/java/fr/xephi/authme/AuthMe.java

```diff
@@ -316,6 +319,8 @@ public void onDisable() {
         // Wait for tasks and close data source
         new TaskCloser(this, database).run();
 
+        spectateLoginService.removeArmorstands();
```

Comment:
Potential null pointer if the plugin crashes before this service is set on L251

Reply:
1. The field on the class is no longer needed (an IDE shows an unused warning)
2. `injector.createIfHasDependencies(SpectateLoginService.class)` might return null, so the issue of a potential null pointer still remains. And I think it would be better to use `injector#getIfAvailable`

Reply:
Done.